src/utils/utils.js

```diff
@@ -4,25 +4,45 @@ const API = axios.create({ baseURL: 'https://ncgames-tsyc.onrender.com/api' })
 export const getAllCategories = () => {
   return API.get("/categories")
     .then(({data}) => {
-      console.log(data.category)
-      return data.category
+      return data.category;
     })
     .catch((error) => {
       // handle error
       console.log(error);
   })
 };
 
-export const getAllReviews = () => {
+export const getAllReviews = (filter) => {
   return API.get("/reviews")
-    .then(({data}) => {
-      console.log(data.Review)
+  .then(({data}) => {
+    if (filter !== ""){
+      const filteredArray = data.Review.filter((review) => review.category === filter)
+      return filteredArray
```

Comment:
I see what you are doing here, however, this is a future ticket. Please delete this for now. Also, you will be able to achieve filtering by adding a query to your api call eg `/api/reviews?category=dexterity` If you reached ticket 11 in your backend, you will be able to do this. If you didn't, there is no need to implement the feature 🙂

Reply:
I see, I tried to fetch the data by API call as I heard someone mention in the seminar. But later I found that "/api/reviews?category=dexterity" doesn't give the data I want as I don't know I have to reach ticket 11 in my backend. This filter here is just for the user to get reviews via "All Category" in Nav bar. ( the logic is "all category" button sets the filter to blank while other categories set the filter to that specific name. If the filter is blank all reviews are shown)